src/Model/Post.php

```diff
@@ -235,6 +249,7 @@ public function init() {
 					return ! empty( $this->post->post_date_gmt ) ? Types::prepare_date_response( $this->post->post_date_gmt ) : null;
 				},
 				'contentRendered' => function() {
+					setup_postdata( $this->post );
```

Comment:
This seems redundant?

Reply:
It's not really though . . . because of Deferred resolution now, there's no guarantee that we will be returning all fields of the same post at the same time. 

We might return the `title` of post 1, 2, 3

Then return `content` of post 1, 2, 3

So each field needs to reset the postdata if it's needed.

Otherwise the globals are all funky.

However, `setup_postdata` is luckily smart enough to read from the cache, so DataLoader is now adding the necessary info to the cache up front, and `setup_postdata` just sets the Globals to already cached data before the fields resolve. 

So we're not costing additional queries like we once were.